source/grid/manifold_lib.cc

```diff
@@ -446,8 +446,8 @@ SphericalManifold<dim, spacedim>::get_tangent_vector(
   const Point<spacedim> &p1,
   const Point<spacedim> &p2) const
 {
-  const double tol = 1e-10;
-  (void)tol;
+  [[maybe_unused]] const double tol = 1e-10;
+  //(void)tol; REMOVED BECAUSE OF [[maybe_unused]]
```

Comment:
There is no point stating that you removed this line: The fact that the line *used* to be part of the code base conveys no useful information to a reader of this code. So let's just remove the old line.